affiliates/Poland.md

```diff
@@ -93,6 +93,9 @@ This method will create or update customer and return customer uuid.
 | signature[timestamp]            | Required <br /> unix timestamp – must be UTC +/- 60 seconds                                                                                         | Timestamp when request is made                             |
 | signature[api_key]              | Required                                                                                                                                      | Api key                                                    |
 | signature[hash]                 | Required                                                                                                                                      | A sha1 concatenation of timestamp, api key and secret key  |
+| agreements.terms_of_service     | Required <br /> Integer [1 - Yes, 0 - No]                                                                                                           | Does customer agree with terms of service?                 |
```

Comment:
it is not `agreements.terms_of_service` but `agreements[terms_of_service]`

Reply:
fixed